lab-matt/package.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
This is a great start, the only thing I'd like you to change is the "scripts" > "test" property.  What this does is specify what to run when a user inputs "npm test" in the terminal (when inside your project root directory) is run the command you specify, since you're using mocha you can specify that property to equal "mocha" rather than "echo \"Error: no test specified\" && exit 1".  Also make sure ISC is the license you want your code to use, it's fine to use that, I'm just reminding because it's the default option and I just want you to make sure to keep that in mind since it's very important.